scylladb/sphinx-scylladb-theme
# rm: fontawesome support

Closes https://github.com/scylladb/sphinx-scylladb-theme/issues/1220

## How to test

1. Clone the PR and build the docs.
2. Check icons render as expected: 
    * http://127.0.0.1:5500/examples/links/
    * http://127.0.0.1:5500/examples/links/
    * http://127.0.0.1:5500/examples/topic-box/
